src/JustEat.StatsD/TimerExtensions.cs

```diff
@@ -214,20 +253,25 @@ public static async Task<T> Time<T>(this IStatsDPublisher publisher, string buck
         /// <param name="publisher">The <see cref="IStatsDPublisher"/> to publish with.</param>
         /// <param name="bucket">The bucket to publish the timer for.</param>
         /// <param name="func">A delegate to a method whose invocation should be timed and result returned.</param>
+        /// <param name="tags">The tags to publish with the timer.</param>
```

Comment:
```suggestion
        /// <param name="tags">The tag(s) to publish with the timer.</param>
```

Reply:
I'm not going to go back and edit the others now, but we should probably annotate this like the below to make clear that tags are not required.

```
The optional tag(s) to publish with the timer.
```